app/src/main/java/com/sunasterisk/weather/screen/weather/WeatherPresenter.kt

```diff
@@ -0,0 +1,34 @@
+package com.sunasterisk.weather.screen.weather
+
+import com.sunasterisk.weather.data.model.Weather
+import com.sunasterisk.weather.data.source.WeatherRepository
+import com.sunasterisk.weather.data.source.remote.OnFetchDataJsonListener
+
+class WeatherPresenter(
+    private val weatherRepository: WeatherRepository
+) : WeatherContract.Presenter {
+    private var viewContract: WeatherContract.View? = null
```

Comment:
nếu class header viết trên nhiều dòng, hãy để trống 1 dòng để phân biệt giữa header và body của class 
```suggestion
) : WeatherContract.Presenter {

    private var view: WeatherContract.View? = null
```